rllib/core/models/configs.py

```diff
@@ -0,0 +1,201 @@
+from dataclasses import dataclass, field
+from typing import List, Callable, Dict
+from ray.rllib.utils.typing import ViewRequirementsDict
+import functools
+
+import gymnasium as gym
+
+from ray.rllib.core.models.base import ModelConfig, Model, Encoder
+from ray.rllib.utils.annotations import ExperimentalAPI
+
+
+@ExperimentalAPI
+def _framework_implemented(torch: bool = True, tf2: bool = True):
```

Comment:
I'm using this decorator as long as I'm developing these models to clearly filter the stuff that does not go with TF.
We can potentially remove it after reachon parity between TF/Torch.